src/components/dtf-chat/overrides.css

```
@@ -70,3 +70,63 @@
+   crisp and inherits the button's text color in every state). */
+.stocks-embedded-chat [data-testid='rc-send'] {
+  font-size: 0;
+}
+
+.stocks-embedded-chat [data-testid='rc-send']::after {
+  content: '';
+  display: inline-block;
+  vertical-align: middle;
+  width: 1rem;
+  height: 1rem;
+  background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)